### DISCUSSION TITLE
Time series panel: preferred Y-axis as a per-query attribute?

### DISCUSSION THREAD

**Discussion by aturetta** [grafana/grafana#40119] [Dashboards, Panels & Visualizations]
in prometheus a single query can generate many dynamic series, and their names are usually dynamically generated from labels.

One can try to move each series manually to the right Y axis, but I think it is not really scalable. You add a pod to the scaling group, and you will get a new series that will mess up your graph.

wouldn't it be easier if the default axis could be assigned per-query?
The current UI to override Y-axis can remain in place, just allow the default value for all the resulting series be set on the query itself.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
